### PR TITLE
Survey Duplication

### PR DESCRIPTION
**Pull request by jeremyyap**
Reference #1020 

- [x] Survey duplication
- [x] Survey factories
- [x] Survey duplication tests

### CODE REVIEW

File: spec/factories/course_survey_questions.rb [hunk 1]
```diff
@@ -1,13 +1,48 @@
 # frozen_string_literal: true
 FactoryGirl.define do
   sequence(:course_survey_question_name) { |n| "Survey Question #{n}" }
-  factory :course_survey_question, class: Course::Survey::Question.name do
+  factory :course_survey_question, class: Course::Survey::Question.name, aliases: [:question] do
```
Comment (weiqingtoh) [inline spec/factories/course_survey_questions.rb]: I think we can use `survey_question` as the alias instead of `question` - might mix up with assessment.

**Comment by jeremyyap** [comment]
@kxmbrian hi boss

**Comment by kxmbrian** [comment]
LGTM!

**Comment by allenwq** [comment]
👍 Good work @jeremyyap !

**Pull request merged**